src/main/java/com/fasterxml/jackson/databind/deser/std/EnumDeserializer.java

```diff
@@ -76,6 +85,22 @@ public EnumDeserializer(EnumResolver byNameResolver, Boolean caseInsensitive)
         _enumDefaultValue = byNameResolver.getDefaultValue();
         _caseInsensitive = caseInsensitive;
         _isFromIntValue = byNameResolver.isFromIntValue();
+        _lookupByEnumNaming = null;
```

Comment:
Maybe instead of implementing this, just call other constructor:

   this(byNameResolver, caseInsensitive, null)